core/options.hpp

```diff
@@ -133,6 +133,8 @@ struct Options
 			 "Camera mode as W:H:bit-depth:packing, where packing is P (packed) or U (unpacked)")
 			("viewfinder-mode", value<std::string>(&viewfinder_mode_string),
 			 "Camera mode for preview as W:H:bit-depth:packing, where packing is P (packed) or U (unpacked)")
+			("buffer-count", value<unsigned int>(&buffer_count)->default_value(0), "overwrite bufferCount config from libcamera for video; raw and still (0 to use default provided by the libcamera implementation)")
+			("viewfinder-buffer-count", value<unsigned int>(&viewfinder_buffer_count)->default_value(0), "overwrite bufferCount config from libcamera  for the preview (0 to use default provided by the libcamera implementation)")
```

Comment:
Could you re-phrase the description a bit.  Something like
```
Number of in-flight requests (and buffers) configured for video; raw; and still.
```

Reply:
Looking at the changes again, the RAW stream buffer count is not updated from the command line argument.